src/gluonts/model/r_forecast/_predictor.py

```diff
@@ -49,57 +50,46 @@
 R -e 'install.packages(c("forecast", "nnfor"), repos="https://cloud.r-project.org")'
 """
 
-SAMPLE_FORECAST_METHODS = ["ets", "arima"]
-QUANTILE_FORECAST_METHODS = ["tbats", "thetaf", "stlar"]
-POINT_FORECAST_METHODS = ["croston", "mlp"]
-SUPPORTED_METHODS = (
-    SAMPLE_FORECAST_METHODS
-    + QUANTILE_FORECAST_METHODS
-    + POINT_FORECAST_METHODS
-)
-
 
 class RForecastPredictor(RepresentablePredictor):
     """
     Wrapper for calling the `R forecast package
```

Comment:
This is not accurate anymore, right? This is now to invoke any of the R wrappers we provide, given that the necessary dependencies are installed